tiles/src/main/java/com/protomaps/basemap/layers/Landuse.java

```diff
@@ -18,61 +19,105 @@ public static void processFeature(SourceFeature sf, FeatureCollector features, S
-          kind = "hospital";
-        } else {
-          kind = "school";
-        }
-      } else if (sf.hasTag("landuse")) {
-        if (sf.hasTag("landuse", "orchard", "farmland", "farmyard")) {
-          kind = "farmland";
-        } else if (sf.hasTag("landuse", "industrial", "brownfield")) {
-          kind = "industrial";
-        } else if (sf.hasTag("landuse", "cemetery")) {
-          kind = "cemetery";
-        }
+      if (sf.hasTag("aeroway", "aerodrome")) {
+        kind = sf.getString("aeroway");
+      } else if (sf.hasTag("amenity",  "university","college", "hospital", "library", "post_office", "school", "townhall")) {
+        kind = sf.getString("amenity");
+      } else if (sf.hasTag("amenity", "cafe")) {
+        kind = sf.getString("amenity");
       } else if (sf.hasTag("highway")) {
         kind = "pedestrian";
+      } else if (sf.hasTag("landuse", "cemetery")) {
+        kind = sf.getString("landuse");
+      } else if (sf.hasTag("landuse", "orchard", "farmland", "farmyard")) {
+        kind = "farmland";
+      } else if (sf.hasTag("landuse", "industrial", "brownfield")) {
+        kind = "industrial";
+      } else if (sf.hasTag("leisure", "golf_course", "marina", "park", "stadium")) {
+        kind = sf.getString("leisure");
       } else if (sf.hasTag("man_made", "bridge")) {
         kind = "pedestrian";
-      } else if (sf.hasTag("aeroway", "aerodrome")) {
-        kind = "aerodrome";
+      } else if (sf.hasTag("shop", "grocery", "supermarket")) {
+        kind = sf.getString("shop");
+      } else if (sf.hasTag("tourism", "attraction", "camp_site", "hotel")) {
+        kind = sf.getString("tourism");
+      } else {
+        // Avoid problem of too many "other" kinds
+        // All these will default to min_zoom of 15
+        // If a more specific min_zoom is needed (or sanitize kind values)
+        // then add new logic in section above
+        if (sf.hasTag("amenity")) {
+          kind = sf.getString("amenity");
+        } else if (sf.hasTag("craft")) {
+          kind = sf.getString("craft");
+        } else if (sf.hasTag("aeroway")) {
+          kind = sf.getString("aeroway");
+        } else if (sf.hasTag("historic")) {
+          kind = sf.getString("historic");
+        } else if (sf.hasTag("landuse")) {
+          kind = sf.getString("landuse");
+        } else if (sf.hasTag("leisure")) {
+          kind = sf.getString("leisure");
+        } else if (sf.hasTag("railway")) {
+          kind = sf.getString("railway");
+        } else if (sf.hasTag("shop")) {
+          kind = sf.getString("shop");
+        } else if (sf.hasTag("tourism")) {
+          kind = sf.getString("tourism");
+        // Boundary is most generic, so place last else we loose out
+        // on nature_reserve detail versus all the protected_area
+        } else if (sf.hasTag("boundary")) {
+          kind = sf.getString("boundary");
+        }
       }
+
+      if( sf.hasTag("boundary", "national_park") &&
```

Comment:
Special case `national_park` kind calcuation.

We should do this for `national_forest`, too...